ui/app/components/transformation-edit.js

```diff
@@ -17,30 +17,12 @@ export default TransformBase.extend({
     this.set('initialRoles', this.model.allowed_roles);
   },
 
-  updateOrCreateRole(role, transformationId, backend) {
-    return this.store
+  async updateOrCreateRole(role, transformationId, backend) {
+    const roleStore = await this.store
```

Comment:
can we call this `roleRecord`? it wasn't clear to me below that this was the record I thought it was maybe the store or something 💭

Reply:
Same thought, I only kept it the same for consistency to compare.